bin/zopen-install

```diff
@@ -398,6 +399,32 @@ handlePackageInstall()
       printf "${NC}${HEADERCOLOR}${BOLD}${name}${NC}:\n ${installCaveat}\n" >> ${caveatsFile}
     fi
 
+    
+    systemPrereqs=$(jq -r '.product.system_prereqs // empty | map(.name) | join(" ")' "${metadataFile}" 2>/dev/null)
+    if ! $bypassPrereqs; then
+      if [ -z "${systemPrereqs}" ]; then
+        systemPrereqs="${systemPrereqs} zos24" # set the min requirest as zos24
+      fi
+      if [ -n "$systemPrereqs" ]; then
+        if [ ! -f "$ZOPEN_SYSTEM_PREREQ_SCRIPT" ]; then
+          printWarning "$ZOPEN_SYSTEM_PREREQ_SCRIPT does not exist. You should upgrade meta. Bypassing prereq check."
```

Comment:
would using -r to test for read-access to the script be a more robust check?  It would provide both a readability and existence check.  
Also, given that the code for doing the pre-req checks will only be from when this PR merges forward, is there a situation where meta is needing to be upgraded?  It would already be at the level that supports prereqs as any code to handle prereqs is not merged yet so the .product.system_prereqs  fields is ignored.  
The warning might be better as an error as the system is not setup correctly so we shouldn't install anything until it has been investigated.  This would require the message changing to something like  "$Z_S_P_S cannot be read. Check file permissions and reinstall the meta package or reinitialize the zopen environment if required. "  - or words to that effect?

Reply:
Makes sense, the warnings could lead to hidden bugs. Updated it